docs/maintain/maintain-guides-how-to-setup-a-validator-with-reverse-proxy.md

```diff
@@ -175,6 +175,38 @@ Nodes will use [libp2p](https://libp2p.io/) as the networking layer to establish
 messages, but uses NGINX as a load balancer which acts as a _first listener_ of the streaming data
 to help balance the load.
 
+Each Polkadot Host node maintains an ED25519 key pair which is used to identify the node. The public
```

Comment:
Good point, I just pulled up `maintain-guides-how-to-setup-a-validator-with-reverse-proxy.md` to add the conditional renderings for both projects and realized this doc is written exclusively in the context of Polkadot.  Mainly references to commands and file names but there is no previous mention of Kusama.

I changed my references of `Polkadot Host` to `host nodes` or `nodes` but we may want to create an additional issue around identifying docs that are exclusive of Kusama references.  It would be an interesting exercise to audit the repo for this condition.  @DrW3RK recently mentioned how the Kusama guide is like a light version of the Polkadot Wiki, this would help to start bridging that gap.

Let me know what you guys think and I can file it if desired.

Reply:
I also notice that some pages that are in the Polkadot wiki are not present on Kusama, and overall the Polkadot wiki is more resourceful. Not sure here what the best strategy is. It would be cool to have one wiki for both and make the whole process of curating content less time-consuming and in general more efficient. Maybe something to discuss internally?

Reply:
One straightforward way to do this is to conditionally render the remaining docs of the Polkadot Wiki to the Kusama Guide. Or, to add placeholders like this https://guide.kusama.network/docs/maintain-guides-how-to-validate-kusama

Certainly an internal discussion would help define the scope.